docs/discovery.md

```diff
@@ -212,7 +212,7 @@ You can contribute a new discovery backend to Swarm. For information on how to d
 
 ## Docker Swarm documentation index
 
-- [User guide](index.md)
-- [Sheduler strategies](/scheduler/strategy.md)
-- [Sheduler filters](/scheduler/filter.md)
-- [Swarm API](/api/swarm-api.md)
+- [User guide](https://docs.docker.com/swarm/)
```

Comment:
@peggyl Thanks for the contribution. These links are actually built with our static file generator.  If you run `make docs` in the docs directory and test locally, you'll find all but the first was resolving.  So, we don't need the full path name here.  

This is correct:
```
- [User guide](/)
- [Sheduler strategies](/scheduler/strategy.md)
- [Sheduler filters](/scheduler/filter.md)
- [Swarm API](/api/swarm-api.md)
```